src/GovUk.Education.ExploreEducationStatistics.Data.Api/Controllers/TableBuilderMetaController.cs

```diff
@@ -13,16 +17,29 @@ namespace GovUk.Education.ExploreEducationStatistics.Data.Api.Controllers
     public class TableBuilderMetaController : ControllerBase
     {
         private readonly ISubjectMetaService _subjectMetaService;
+        private readonly ICacheKeyService _cacheKeyService;
 
-        public TableBuilderMetaController(ISubjectMetaService subjectMetaService)
+        public TableBuilderMetaController(
+            ISubjectMetaService subjectMetaService, 
+            ICacheKeyService cacheKeyService)
         {
             _subjectMetaService = subjectMetaService;
+            _cacheKeyService = cacheKeyService;
         }
 
         [HttpGet("subject/{subjectId}")]
         public Task<ActionResult<SubjectMetaViewModel>> GetSubjectMetaAsync(Guid subjectId)
         {
-            return _subjectMetaService.GetSubjectMeta(subjectId).HandleFailuresOrOk();
+            return _cacheKeyService
```

Comment:
For this PR I have cached the lists of SubjectViewModels as a `subjects.json` file in our cache, one per Release.  Makes a big difference.  In the future it might be better to cache at a per-Subject level but currently, with Services potentially shareable between Admin and Data APIs, we don't want to accidentally invoke caching from the Admin side with a shareable service, hence I have been caching things at the Controller level in the Data API project to avoid this possibility.

Possibly we have to look at ways of allowing different API projects to configure what caching they allow and what they don't when using a shared Service.